src/WebJobs.Script.Grpc/GrpcServiceCollectionsExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.WebJobs.Script.Grpc
+{
+    public static class GrpcServiceCollectionsExtensions
+    {
+        public static IServiceCollection AddGrpc(this IServiceCollection services)
+        {
+            services.AddSingleton<FunctionRpc.FunctionRpcBase, FunctionRpcService>();
+            services.AddSingleton<IRpcServer, GrpcServer>();
+            services.AddSingleton<IRpcWorkerProcessFactory, RpcWorkerProcessFactory>();
+            services.AddSingleton<IRpcWorkerChannelFactory, RpcWorkerChannelFactory>();
```

Comment:
I think I should refactor this such that in here, we only register `IRpcWorkerChannelFactory` (as discussed offline) and maybe also move `RpcInitializationService` here.

That way `WebJobs.Script.Grpc` only exports a channel factory and manages whatever service it needs internally.